src/containers/AccountModal/forms/ResetPassword.tsx

```diff
@@ -66,8 +68,9 @@ const ResetPassword: React.FC<Prop> = ({ type }: Prop) => {
     const resetUrl = `${window.location.origin}/?u=edit-password`;
 
     try {
-      await resetPassword(formData.email, resetUrl);
-      navigate(addQueryParam(location, 'u', 'send-confirmation'));
+      await resetPassword({ customerEmail: formData.email, resetUrl });
```

Comment:
I'm confused why this is calling reset password and then going to edit-password.  I thought edit-password was only accessible via email links in the password reset email?  What does it mean to edit a password after resetting the password?

Reply:
Yes, you are right, `edit-password` is only accessible via email links in the password reset email. But this makes sense to me because first, you need to reset the password and get a temporary password or token in the email, then you need to update the password